### DISCUSSION TITLE
disableBackdropClick

### DISCUSSION THREAD

**Discussion by utiq** [tailwindlabs/headlessui#875] [Ideas]
When I click the backdrop of a Dialog the Dialog closes. I'm not sure if there is an option to prevent that.
It would be nice to add something like:
`<Dialog disableBackdropClick />`

**Comment by RobinMalfait** [comment]
Hey! Thank you for your question!
Much appreciated! 🙏

If you don't want the `Dialog.Overlay` to be clickable, then you can use a plain html element, like a `div`.
